payments-core/src/main/java/com/stripe/android/networking/StripeRepository.kt

```diff
@@ -431,4 +433,10 @@ abstract class StripeRepository {
         paymentMethodTypes: Set<String>,
         requestOptions: ApiRequest.Options
     ): ConsumerPaymentDetails?
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    abstract suspend fun paymentIntentLinkAccountSession(
```

Comment:
Can we name this to `createPaymentIntentLinkAccountSession` so it is clear that its a POST?